lib/hexpm_web/templates/package/audit_logs.html.eex

```diff
@@ -0,0 +1,32 @@
+<h2 class="package-title">
+  <%= ViewHelpers.package_name(@package) %>
+  <span class="version">Recent Activities</span>
+</h2>
```

Comment:
We should use the same style header we have for `templates/version/index.html.eex`.

Reply:
updated in bcea77d0b86710f417b917862ad38adf6e90cbff